### DISCUSSION TITLE
change log

### DISCUSSION THREAD

**Discussion by autoca** [freeplane/freeplane#774] [Help] [answered]
Hi all,

where can I find the change log for the latest beta / preview version again?
I thought I had seen it for 05 before, but now for 06 I can't find it anymore.

**Comment by nnako** [comment]
https://sourceforge.net/projects/freeplane/files/freeplane%20preview/readme.txt/download

**Comment by autoca** [comment, marked as answer]
Thank you, nnako,
although your information is flawed, since the readme is extremely old, you helped me to see better:
It is https://sourceforge.net/projects/freeplane/files/freeplane%20preview/history_en-1.10.5-pre04.txt/download
But obviously, at the moment it only includes changes up to pre04, not even 05 as I thought before.

**Reply by dpolivaev** [reply]
The most truth is contained in the commit messages, the history file is just some extract.

**Reply by autoca** [reply]
But how do I differentiate the version in the commit messages?
Do I have to compare the date of each commit with the date of the pre-version?

**Reply by dpolivaev** [reply]
All released versions including the previews have tags ( at least after I remember that I should push them)

**Reply by autoca** [reply]
Is it possible to see the tags also in the commits overview?
I only see them in details view, which is cumbersome for visually scanning all commits if I have to go into details for each of them until I find the "version change" and then having to remember which commits are the first and last one of that version.

Tag in details view:
![grafik](https://user-images.githubusercontent.com/37442404/198031927-7dfcdc54-5213-45bd-9908-671290df822a.png)

No tag in overview:
![grafik](https://user-images.githubusercontent.com/37442404/198030732-7bd4b595-2960-49e6-b622-cc3b9bbc5ddd.png)

**Reply by dpolivaev** [reply]
I don't know.

**Reply by autoca** [reply]
ok

**Reply by macmarrum** [reply]
When I use the `gitk` command (it's installed with git for Windows), I can see the tags

![2022-10-26_18,08,11](https://user-images.githubusercontent.com/7197131/198078238-c566cf95-fb79-4942-851a-a703bc88e1be.png)

**Reply by autoca** [reply]
ok, but I want to see it on github
